tests/ScopeTest.php

```diff
@@ -173,7 +173,7 @@ public function testContitionToWords()
 
         $country = clone $this->country;
 
-        $country->addCondition('Users/#');
+        $country->addCondition('Users/#', '>', 0);
```

Comment:
needed for MSSQL, see https://dba.stackexchange.com/questions/275313/can-an-expression-be-used-as-an-expression-as-well-as-a-predicate